src/SingleContainer/Orchestrator/NetworkPerspective.Sync.Orchestrator.Application/Scheduler/SecretRotation/SecretRotationScheduler.cs

```diff
@@ -29,7 +29,7 @@ public SecretRotationScheduler(IQuartzSchedulerFactory schedulerFactory, IOption
 
     protected override async Task ExecuteAsync(CancellationToken stoppingToken)
     {
-        //await Task.Delay(TimeSpan.FromMinutes(5));
+        await Task.Delay(20000);
```

Comment:
To chyba nie o to chodziło. Zależało mi na tym żeby cały quartz startował 5 min po starcie aplikacji (wartość domyślna ale konfigurowalna a app.settings żeby dało się ). To daje czas na podłączenie się workerów, ale też uniknięcie konfliktów jeśli dwa orchestratory są uruchomione równolegle ze względu na aktualizację. BTW: zadania synchronizacji również powinny czekać na podłączenie się workerów. Poniższe ustawienie to zapewnia.

w orchestrator/schedular/sync/ServiceCollectionExtensions.cs linia 50
        services.AddQuartzHostedService(q =>
        {
            q.WaitForJobsToComplete = true;
            q.StartDelay = TimeSpan.FromSeconds(10);  <==== tutaj
        });

Reply:
Zgadza sie, ale moja intencja nie bylo rozwiazanie problemu zawartego w tickecie NP-1260, to zadanie oberwało rykoszetem...